js/ftx.js

```diff
@@ -1925,6 +1928,39 @@ module.exports = class ftx extends Exchange {
         return result;
     }
 
+    async cancelOrders (ids, symbol = undefined, params = {}) {
+        /**
+         * @method
+         * @name ftx#cancelOrders
+         * @description cancel multiple orders
+         * @param {[string]} ids order ids
+         * @param {string|undefined} symbol not used by ftx cancelOrders ()
+         * @param {object} params extra parameters specific to the ftx api endpoint
+         * @returns {object} raw - a list of order ids queued for cancelation
+         */
+        await this.loadMarkets ();
+        // https://docs.ccxt.com/en/latest/manual.html#user-defined-clientorderid
+        const clientOrderIds = this.safeValue2 (params, 'client_order_ids', 'clientOrderIds');
```

Comment:
@AlgoQ, minor thing, we usually support two different formats for each parameter, the ccxt-unified term and the exchange-specific one, in this case, they're the same `clientOrdersIds`, so we don't need `client_orders_ids` here